pyramid/scripts/pcreate.py

```diff
@@ -21,73 +22,76 @@ def main(argv=sys.argv, quiet=False):
-                           'package.')
+    verbosity = 1  # required
+    parser = argparse.ArgumentParser(
+        usage="%(prog)s [options] -s <scaffold> output_directory",
+        description="Render Pyramid scaffolding to an output directory")
+    parser.add_argument('-s', '--scaffold',
+                        dest='scaffold_name',
+                        action='append',
+                        help=("Add a scaffold to the create process "
+                              "(multiple -s args accepted)"))
+    parser.add_argument('-t', '--template',
+                        dest='scaffold_name',
+                        action='append',
+                        help=('A backwards compatibility alias for '
+                              '-s/--scaffold.  Add a scaffold to the '
+                              'create process (multiple -t args accepted)'))
+    parser.add_argument('-l', '--list',
+                        dest='list',
+                        action='store_true',
+                        help="List all available scaffold names")
+    parser.add_argument('--list-templates',
+                        dest='list',
+                        action='store_true',
+                        help=("A backwards compatibility alias for -l/--list. "
+                              "List all available scaffold names."))
+    parser.add_argument('--package-name',
+                        dest='package_name',
+                        action='store',
+                        help='Package name to use. The name provided is '
+                             'assumed to be a valid Python package name, and '
+                             'will not be validated. By default the package '
+                             'name is derived from the value of '
+                             'output_directory.')
+    parser.add_argument('--simulate',
+                        dest='simulate',
+                        action='store_true',
+                        help='Simulate but do no work')
+    parser.add_argument('--overwrite',
+                        dest='overwrite',
+                        action='store_true',
+                        help='Always overwrite')
+    parser.add_argument('--interactive',
+                        dest='interactive',
+                        action='store_true',
+                        help='When a file would be overwritten, interrogate '
+                             '(this is the default, but you may specify it to '
+                             'override --overwrite)')
+    parser.add_argument('--ignore-conflicting-name',
+                        dest='force_bad_name',
+                        action='store_true',
+                        default=False,
+                        help='Do create a project even if the chosen name '
+                             'is the name of an already existing / importable '
+                             'package.')
+    parser.add_argument('output_directory',
```

Comment:
You'll need to make this optional if you want the program to behave the same as before... otherwise argparse will helpfully sys.exit() and let the user know they need to specify  an output directory...